### PR TITLE
Adding gradient accum to torchvision ic

### PR DESCRIPTION
**Pull request by corey-nm**

### CODE REVIEW

**Review by rahul-tuli — APPROVED** [review]

File: src/sparseml/pytorch/torchvision/train.py [hunk 1]
```diff
@@ -45,6 +45,11 @@ def train_one_epoch(
     metric_logger.add_meter("lr", utils.SmoothedValue(window_size=1, fmt="{value}"))
     metric_logger.add_meter("img/s", utils.SmoothedValue(window_size=10, fmt="{value}"))
 
+    accum = 0
```
Comment (KSGulin) [inline src/sparseml/pytorch/torchvision/train.py:48]: nite: make name more verbose. e.g. `steps_accumulated`

**Review by KSGulin — COMMENTED** [review]
Just one nit, LGTM in general though

File: src/sparseml/pytorch/torchvision/train.py (hunk 1 above)
Comment (corey-nm) [reply src/sparseml/pytorch/torchvision/train.py]: changed!
